lib/libm/erf_lgamma.c

```diff
@@ -23,6 +23,7 @@
  *
  */
 
+#include "py/mpconfig.h"
```

Comment:
Hmm.. this is not great, the libm library shouldn't depend on this header.  Is there another way around this?  Maybe pass through a definition of `MP_FALLTHROUGH` in the makefile?

Reply:
libm isn't included in in the scope of this PR (?) so this file could just as well be left untouched? Or as another alternative: excluded from being compiled with this particular warning enabled. Just thinking that it would be not too concistent and possibly confusing to have nearly all `MP_XXX` complier-specifics in mpconfig.h but not this one.

Reply:
If possible my preference would be to not touch this file.  But yet another alternative would be to define a separate fallthrough macro in `libm.h`, eg `LIBM_FALLTHROUGH`.

Reply:
Agree. I guess I must have misunderstood @stinos request to replace all fallthrough comments without extra info with the macro.

Reply:
Indded sorry if that wasn't clear, I meant 'all the ones you changed so far' :)